servicetalk-dns-discovery-netty/src/main/java/io/servicetalk/dns/discovery/netty/DefaultDnsServiceDiscovererBuilder.java

```diff
@@ -44,10 +49,43 @@
  */
 @Deprecated // FIXME: 0.43 - make package private
 public final class DefaultDnsServiceDiscovererBuilder implements DnsServiceDiscovererBuilder {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultDnsServiceDiscovererBuilder.class);
+
+    /**
+     * @deprecated This system property is introduced temporarily as a way for users to skip binding and revert the
+     * preexisting behavior.
+     */
+    @Deprecated // FIXME: 0.43 - consider removing this system property
+    private static final String SKIP_BINDING_PROPERTY = "io.servicetalk.dns.discovery.netty.skipBinding";
```

Comment:
This is a quick escape way in case binding to the wildcard address causes any issues for users.

Reply:
I'm not seeing where `SKIP_BINDING` is used. Did you intend to test it when constructing the `DefaultDnsClient` or when defining `DEFAULT_LOCAL_ADDRESS`? Presuming so, which do you prefer?

Reply:
Good catch! I forget to use it 🙈